packages/UsageAnalysis/files/Test Track/Viewers/World cloud.md

```diff
@@ -0,0 +1,17 @@
+### World cloud
+
+1. Open the **SPGI** dataset.
```

Comment:
Link dataset to the test case

Reply:
added   "datasets": ["System:DemoFiles/SPGI.csv"]